doc/operation-and-maintenance/Logging-&-monitoring.md

```diff
@@ -52,33 +52,23 @@ To monitor MongooseIM during load testing, we recommend the following open sourc
 - [collectd](http://collectd.org/) is a daemon running on the monitored nodes capturing data related to CPU and Memory usage, IO etc.
 
 
-### Built-in Exometer reporters
-
-MongooseIM uses the Exometer libary for collecting the metrics. 
-Exometer has many build-in reporters that can send metrics to external services like:
-
-* graphite
-* amqp
-* statsd
-* snmp
-* opentsdb
+### Plug-in Exometer reporters
 
+MongooseIM uses [a fork of Exometer library](https://github.com/esl/exometer_core) for collecting metrics.
+Exometer has many plug-in reporters that can send metrics to external services. We maintain [exometer_report_graphite](https://github.com/esl/exometer_report_graphite) and [exometer_report_statsd](https://github.com/esl/exometer_report_statsd) for Graphite and StatsD respectively.
 It is possible to enable them in MoongooseIM via the `app.config` file.
 The file sits next to the `ejabberd.cfg` file in the `rel/files` and `_REL_DIR_/etc` directories.
-For more details, please visit the Exometer's project page: [Exometer](https://github.com/Feuerlabs/exometer).
 
-**Note that we are using the 1.2.1 version with our patches.**
-
-Below you can find a sample configuration. 
+Below you can find a sample configuration.
 It shows setting up a reporter connecting to graphite running on localhost.
 
 You can see an additional option not listed in the Exometer docs - `mongooseim_report_interval`.
-That option sets the metrics resolution: how often Exometer gathers and sends metrics through reporters. 
+That option sets the metrics resolution: how often Exometer gathers and sends metrics through reporters.
```

Comment:
The option sets the metric resolution: how often Exometer gathers and sends the metrics through reporters.

The option sets the metrics' resolution: how often Exometer gathers and sends the metrics through reporters.